cmd/controller/app/options/options.go

```diff
@@ -223,10 +224,10 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"The duration the clients should wait between attempting acquisition and renewal "+
 		"of a leadership. This is only applicable if leader election is enabled.")
 
-	fs.StringSliceVar(&s.EnabledControllers, "controllers", defaultEnabledControllers, ""+
-		"The set of controllers to enable.")
-	fs.StringSliceVar(&s.DisabledControllers, "disabled-controllers", defaultDisabledControllers, ""+
-		"The set of controllers to disable. Supersedes --controllers.")
+	fs.StringSliceVar(&s.EnabledControllers, "controllers", defaultEnabledControllers, fmt.Sprintf(""+
+		"A list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller "+
+		"named 'foo', '-foo' disables the controller named 'foo'.\nAll controllers: %s",
```

Comment:
Suggestion: maybe remove the single quotes from the example values/replace with double quotes as it seem to suggest that values passed to the flag should be single-quoted? 
I noticed whilst playing with this that if the value for this flag is single-quoted in the deployment manifest it doesn't work 
```
cert-manager "msg"="error executing command" "error"="error validating options: validation failed for '--controllers': [\"'ingress-shim'\" is not in the list of known controllers]"
```

(double quotes/no quotes work ok)

Reply:
Suggestion: Maybe it's worth adding an example for where you want to disable just one controller, but want to keep the rest i.e `--controllers=*,-certificaterequests-approver` ?

I guess people might not expect that passing  `--controllers=-certificaterequests-approver` will in fact disable all the controllers.
 Not strictly necessary though I guess as it can be understood by reading the flag description properly (which I did not at first 😄 )